### ISSUE TITLE
is there a general 4

### ISSUE THREAD

**Issue by Elfish-beaker** [userunknown358/kristysonleaks#1]
I see general 3 and 5 is there an general 4 or is that lost as well?

**Comment by userunknown358** [comment]
Unfortunately, General 4 is lost. But there are some people that have the missing files

**Comment by Elfish-beaker** [comment]
> Unfortunately, General 4 is lost. But there are some people that have the missing files
 alrighty thought id ask. Lots of weird stuff being said by Kris...

**Comment by userunknown358** [comment]
> > alrighty thought id ask. Lots of weird stuff being said by Kris...

All of this I believe should be in General_Full.zip

**Comment by Elfish-beaker** [comment]
> All of this I believe should be in General_Full.zip
I cant really view all of General_Full.zip the page dies and says out of memory gonna have to edit the HTML and splice it 
https://github.com/user-attachments/assets/d32129fc-1ad2-4452-819b-9a71de7f72b0

**Comment by userunknown358** [comment]
> > I cant really view all of General_Full.zip the page dies and says out of memory gonna have to edit the HTML and splice it
> > https://github.com/user-attachments/assets/d32129fc-1ad2-4452-819b-9a71de7f72b0

Yes, you can splice the HTML pages. You can also use other HTML viewers besides Google Chrome.
